math/gnuplot/Portfile

```diff
@@ -132,7 +132,7 @@ variant old_bitmap_terminals description "Enable PBM (Portable Bit Map) and othe
     configure.args-replace      --without-bitmap-terminals --with-bitmap-terminals
 }
 
-default_variants                +luaterm +pangocairo +x11
+default_variants                +luaterm +pangocairo +qt
```

Comment:
This is not a good default on recent systems because using qt4 forces an x86_64 build. You also can't unconditionally set +qt because the user might have already asked for +qt5, which will conflict. I took care of these problems in 9f48f14da.

Reply:
Thanks, @jmroot . Beautiful solution!